### PR TITLE
rename iprepd whitelist to exempt

### PR DESCRIPTION
**Pull request by kkleemola**
The other half of #432 

Note: the slash commands for FoxSecBot have changed so we'll need to update them.

### CODE REVIEW

**Comment by ajvb** [comment]
@kkleemola What is the migration process/steps for this?

File: src/main/java/com/mozilla/secops/postprocessing/PostProcessing.java [hunk 1]
```diff
@@ -361,7 +361,7 @@ private Alert createAlert(Alert sourceAlert, Watchlist.WatchlistEntry entry, Key
       a.addMetadata(AlertMeta.Key.MATCHED_METADATA_VALUE, k.value);
       a.addMetadata(AlertMeta.Key.MATCHED_TYPE, entry.getType());
       a.addMetadata(AlertMeta.Key.MATCHED_OBJECT, entry.getObject());
-      a.addMetadata(AlertMeta.Key.WHITELISTED_ENTRY_CREATED_BY, entry.getCreatedBy());
+      a.addMetadata(AlertMeta.Key.EXEMPTION_CREATED_BY, entry.getCreatedBy());
```
Comment (ajvb) [inline src/main/java/com/mozilla/secops/postprocessing/PostProcessing.java:364]: Oh hm. I believe this was a bug. This should be `WATCHLIST_CREATED_BY`

**Comment by kkleemola** [comment]
I believe the only other migration step is copying the datastore entries to the new namespace. But because there's only three items in there I was just going to manually add those ips with the slack command. (We could also write a small script to copy datastore entries to a different namespace, but not sure it's worth it for 3 items)

File: src/main/java/com/mozilla/secops/postprocessing/PostProcessing.java (hunk 1 above)
Comment (kkleemola) [reply src/main/java/com/mozilla/secops/postprocessing/PostProcessing.java]: Nice catch. Looks like that was only used by watchlist so I removed exemption and added watchlist.

**Pull request merged**